code/modules/species/species_shapeshift.dm

```diff
@@ -13,57 +13,64 @@ var/list/wrapped_species_by_ref = list()
 
 	var/list/valid_transform_species = list()
 	var/monochromatic
-	var/default_form = SPECIES_HUMAN
+	var/default_form
+
+/datum/species/shapeshifter/New()
+	default_form = GLOB.using_map.default_species
+	valid_transform_species |= default_form
+	..()
 
```

Comment:
shapeshifters don't fake their root_species_name is that intended?

Reply:
Fixed.